crowbar_framework/config/initializers/installer.rb

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2015, SUSE LINUX Products GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Rails.application.config.tap do |config|
+  unless File.exist?(Rails.root.join(".crowbar-installed-ok")) || ENV["RAILS_ENV"] == "test"
+    ENV["CROWBAR_MODE"] = "installer"
```

Comment:
So this env var is only used within the before filter, why do we need to define it than?